examples/dampened_oscillator/dampened_oscillator_declarative.py

```diff
@@ -0,0 +1,133 @@
+from numerous.multiphysics.equation_decorators import Equation
+from numerous.multiphysics.equation_base import EquationBase
+from numerous.engine.system.item import Item
+from numerous.engine.system import Subsystem, ConnectorTwoWay, ItemsStructure
+from numerous.declarative.specification import ScopeSpec, ItemsSpec, Module, EquationSpec
+from numerous.declarative.variables import Parameter, Constant, State
+from numerous.declarative.bus import Connector, create_connections, get_value_for, set_value_from
```

Comment:
Do these new methods/classes have documentation already?

Reply:
They have a very basic docstring.

Reply:
can this docstring be expanded beyond the basic one?